settings/LoanPolicy/components/DeleteEntry/DeleteEntry.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { Button } from '@folio/stripes/components';
+import { IfPermission } from '@folio/stripes/core';
+
+import { DeleteConfirmationModal } from '../../../components';
+
+class DeleteEntity extends React.Component {
+  static propTypes = {
+    isOpen: PropTypes.bool.isRequired,
+    perm: PropTypes.string.isRequired,
+    policyName: PropTypes.string.isRequired,
+    deleteEntityKey: PropTypes.string.isRequired,
+    initialValues: PropTypes.object,
```

Comment:
Please add isRequired to errors property or defaultProps if this property is not required